languages/python/exercises/concept/currency-exchange/exchange_test.py

```diff
@@ -0,0 +1,68 @@
+import unittest
+from exchange import *
+
+
+class TestExchange(unittest.TestCase):
+    
+    # Problem 1
+    def test_estimate_value(self):
```

Comment:
Each unit tests can be simplified and spitted in smaller tests, each test should test one think and preferably have only one assert statement.

An example would be:
```
 def test_estimate_value_stronger_currency(self):
    estimate = int(estimate_value(100000, 0.84))
    self.assertEqual(estimate, 119047)
```

Reply:
@valentin-p - I think in this context, it is fine to parameterize the tests, provided that PyTest is told to run them as `sub-tests` (which is being done here) so that any potential failures are output individually.

Reply:
IMHO, it makes the test file easier to read and shorter, since the variations for these are an equivalency class.